code/datums/achievements/achievements.dm

```diff
@@ -40,3 +40,10 @@
 	desc = "Kill yourself using the nuclear authentication disk"
 	id = 7
 	hidden = TRUE
+
+
+/datum/achievement/dab
+	name = "King of Dab"
+	desc = "Hit the ULTIMATE dab"
+	id = 420
+	hidden = FALSE
```

Comment:
Number should line up and also should be hidden

Reply:
Thanks for your input mentor.
Both of your suggestions are false.

The ID does not need to line up it just needs to be a unique number. 
And this achievement isn't hidden so it is set to FALSE.

I can see where the confusion is with hidden as its a default value so I'll probably just remove the flag since it is redundant to make our code pretty but otherwise, I am not changing the flag on this achievement to true unless people throw a big stink over it. 
The changelog itself that everyone gets access to directly states that this achievement exists.

Reply:
I went ahead and committed your second suggestion. 
Thank you for saving our codebase from that redundant variable.

Reply:
Numbers do have to line up for organizational purposes

Reply:
We had this discussion about half a week ago. 
Our HC @nichlas0010 quite literally said that as a long as the ID is unique he does not care what the ID itself is even if it is out of order.

Reply:
Seems @nichlas0010 is wrong